src/views/error-views/dev-dot-404/index.tsx

```diff
@@ -1,37 +1,39 @@
-import React, { ReactElement } from 'react'
-import { ProductSlug } from 'types/products'
+import React, { ReactElement, useMemo } from 'react'
```

Comment:
⛏️  Nitpick: Not 100% sure, but doesn't look like `useMemo` is being used here!

```suggestion
import React, { ReactElement } from 'react'
```